pkg/model/projects.go

```diff
@@ -53,13 +53,13 @@ func (e ProjectStatus) String() string {
 type Project struct {
 	BaseModel
 
-	Name      string
-	Type      ProjectType
-	StartDate *time.Time
-	EndDate   *time.Time
-	Status    ProjectStatus
-	Members   []ProjectMember
-	Heads     []ProjectHead
+	Name      string          `json:"name"`
```

Comment:
Why do we need json here?

Reply:
model.Project will return in view `EmployeeData`. Should add `JSON` tag to convert it to camelcase

Reply:
can we define a new struct in view package?

Reply:
okay